src/app/views/interactives/components/ReactTable.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect } from "react";
+import { useTable, usePagination } from "react-table";
+import { store } from "../../../config/store";
+// from https://react-table.tanstack.com/docs/overview
+
+import { Link } from "react-router";
+
+const rootPath = store.getState().state.rootPath;
+
+function Table({ columns, data }) {
+    // Use the state and functions returned from useTable to build your UI
```

Comment:
There is already a component for listing items, can it not be used here?
https://github.com/ONSdigital/florence/blob/develop/src/app/components/simple-selectable-list/SimpleSelectableList.jsx